gradio/chat_interface.py

```diff
@@ -606,11 +611,8 @@ def _append_multimodal_history(
         else:
             for x in message.get("files", []):
                 if isinstance(x, dict):
-                    history.append(  # type: ignore
-                        {"role": "user", "content": cast(FileDataDict, x)}  # type: ignore
-                    )
-                else:
-                    history.append({"role": "user", "content": (x,)})  # type: ignore
+                    x = x.get("path")
```

Comment:
Files should always be stored as tuples in the history, for consistency and as per the documentation.
<img width="867" alt="image" src="https://github.com/user-attachments/assets/e33a61ed-e8fb-465b-8c52-9c9aa8e56f62">